pagopa/io-app
# chore: [LLK-62] Added signBody config

## Short description
Added signBody config.

## List of changes proposed in this pull request
- ts/features/lollipop/index.ts: added signBody option to LollipopConfig.
- ts/features/lollipop/utils/fetch.ts: added signBody parameter handling.

## How to test
Try making an lp request with a body and check if the body is signed, depending on whether or not the signBody parameter is true.
